opengate/actors/coincidences.py

```diff
@@ -1,6 +1,67 @@
 import awkward as ak
 from tqdm import tqdm
 from ..exception import fatal
+from .base import ActorBase
+
+
+class Coincidences(ActorBase):
```

Comment:
Add process_cls(...) at the end of the file
eg: https://github.com/OpenGATE/opengate/blob/master/opengate/actors/arfactors.py#L416

Reply:
Done